### ISSUE TITLE
[Bug] datax cannot be used after deploying through docker compose in centos environment

### ISSUE THREAD

**Issue by codergh** [apache/dolphinscheduler#12882]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

环境：centos
版本：3.1.1
部署工具：docker compose
部署脚本：
```version: "3.8"

services:

  dolphinscheduler-zookeeper:
    image: bitnami/zookeeper:3.6.2
    user: root
    profiles: ["all"]
    environment:
      ALLOW_ANONYMOUS_LOGIN: "yes"
      ZOO_4LW_COMMANDS_WHITELIST: srvr,ruok,wchs,cons
    volumes:
      - /data/zookeeper:/bitnami/zookeeper:rw
    ports:
      - '2181:2181'
    healthcheck:
      test: ["CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/2181"]
      interval: 5s
      timeout: 60s
      retries: 120
    networks:
      - microservice-network

  dolphinscheduler-schema-initializer:
    image: apache/dolphinscheduler-tools:3.1.0
    env_file: .env
    profiles: ["schema"]
    command: [ tools/bin/upgrade-schema.sh ]
    volumes:
      - /data/dolphin/logs:/opt/dolphinscheduler/logs
      - /data/dolphin/shared:/opt/soft
      - /data/dolphin/resource:/dolphinscheduler
    networks:
      - microservice-network

  dolphinscheduler-api:
    image: apache/dolphinscheduler-api:3.1.0
    ports:
      - "12345:12345"
      - "25333:25333"
    profiles: ["all"]
    env_file: .env
    healthcheck:
      test: [ "CMD", "curl", "http://localhost:12345/dolphinscheduler/actuator/health" ]
      interval: 30s
      timeout: 5s
      retries: 3
    volumes:
      - /data/dolphin/logs:/opt/dolphinscheduler/logs
      - /data/dolphin/shared:/opt/soft
      - /data/dolphin/resource:/dolphinscheduler
    networks:
      - microservice-network

  dolphinscheduler-alert:
    image: apache/dolphinscheduler-alert-server:3.1.0
    profiles: ["all"]
    env_file: .env
    healthcheck:
      test: [ "CMD", "curl", "http://localhost:50053/actuator/health" ]
      interval: 30s
      timeout: 5s
      retries: 3
    volumes:
      - /data/dolphin/logs:/opt/dolphinscheduler/logs
    networks:
      - microservice-network

  dolphinscheduler-master:
    image: apache/dolphinscheduler-master:3.1.0
    profiles: ["all"]
    env_file: .env
    healthcheck:
      test: [ "CMD", "curl", "http://localhost:5679/actuator/health" ]
      interval: 30s
      timeout: 5s
      retries: 3
    volumes:
      - /data/dolphin/logs:/opt/dolphinscheduler/logs
      - /data/dolphin/shared:/opt/soft
    networks:
      - microservice-network

  dolphinscheduler-worker:
    image: apache/dolphinscheduler-worker:3.1.0
    profiles: ["all"]
    env_file: .env
    healthcheck:
      test: [ "CMD", "curl", "http://localhost:1235/actuator/health" ]
      interval: 30s
      timeout: 5s
      retries: 3
    volumes:
      - /data/dolphin/data:/tmp/dolphinscheduler
      - /data/dolphin/logs:/opt/dolphinscheduler/logs
      - /data/dolphin/shared:/opt/soft
      - /data/dolphin/resource:/dolphinscheduler
    networks:
      - microservice-network

networks:
  microservice-network:
    name: microservice-network
    external: true
    driver: bridge
```

问题：
无法使用datax，
表现：
<img width="862" alt="image" src="https://user-images.githubusercontent.com/16535355/201528381-86c59eff-ae0e-4fe7-8926-f2a0d8e12be4.png">
`在执行command时,无法识别source命令，导致datax目录报错，找不到datax.py`

### 具体原因大概是下面这段代码
<img width="445" alt="image" src="https://user-images.githubusercontent.com/16535355/201528456-6bc8d003-53c9-4fce-b38c-913f64e16723.png">

### What you expected to happen

希望能解决这个问题

### How to reproduce

fix bug

### Version

3.1.x

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by codergh** [comment]
I updated the question

**Comment by simsicon** [comment]
It seems you are running 3.1.0, can you try it with 3.1.1? This issue should be fixed by this PR https://github.com/apache/dolphinscheduler/pull/12180 , and merged into 3.1.1

**Comment by SbloodyS** [comment]
The basic image does not contain third-party components. Please build it yourself. I'll close this issue. @codergh
